.circleci/config.yml

```diff
@@ -27,6 +27,9 @@ executors:
         environment:
           TERM: xterm
 
+orbs:
+  compare-url: iynere/compare-url@0.4.9
```

Comment:
They recommend using this orb to reconstruct the url and use it as...

Reply:
I see, ok